asabiris/output/msteams/service.py

```diff
@@ -33,18 +33,23 @@ def __init__(self, app, service_name="MSTeamsOutputService"):
 
         self.ConfigService = app.get_service("TenantConfigExtractionService")
 
-    async def send(self, body, tenant):
+    async def send(self, body):
         """
         Sends a message to MS Teams with a provided body content.
         """
         webhook_url = self.TeamsWebhookUrl
 
+        try:
+            effective_tenant = asab.contextvars.Tenant.get()
+        except LookupError:
+            effective_tenant = None
+
         # If tenant-specific MS Teams configuration is available, fetch the webhook URL
-        if tenant and self.ConfigService is not None:
+        if effective_tenant and self.ConfigService is not None:
```

Comment:
what if there is an `effective_tenant` while `self.ConfigService is None`? does that ever happen? if so, shouldnt you at least log an error?

Reply:
(same for slack service)